src/main/java/com/techreturners/weatheripe/WeathipeApplication.java

```diff
@@ -1,16 +1,26 @@
 package com.techreturners.weatheripe;
 
-import com.techreturners.weatheripe.configuration.SecretConfiguration;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.reactive.function.client.ExchangeStrategies;
+import org.springframework.web.reactive.function.client.WebClient;
 
 @SpringBootApplication
-@EnableConfigurationProperties(SecretConfiguration.class)
```

Comment:
@EnableConfigurationProperties(SecretConfiguration.class) 

this needs to be here. I think its being overridden by your changes